test/jdk/java/foreign/TestSegmentOffsetOf.java

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
We used to have a test for MemoryAddress.segmentOffset...
https://github.com/openjdk/jdk/blob/master/test/jdk/java/foreign/TestRebase.java
I wonder if that old test can be put to new use?